modules/analytics/con-telemetry-data-collection-and-analysis.adoc

```diff
@@ -0,0 +1,34 @@
+[id="con-telemetry-data-collection-and-analysis_{context}"]
+= Telemetry data collection and analysis
+
+The telemetry data collection feature helps in collecting and analyzing the telemetry data to improve your experience with {product}. This feature is enabled by default.
+
+{company-name} collects and analyzes the following data:
+
+Web Analytics::
+
+* Web Analytics use the Segment tool.
+
+* It is the tracking of user behavior and interactions with {product}.
+
+Specifically, it tracks the following:
+
+* Events of page visits and clicks on links or buttons.
+* System-related information, for example, locale, time zone, user agent including browser and operating system details.
+* Page-related information, for example, title, category, extension name, URL, path, referrer, and search parameters.
+* Anonymized IP addresses, recorded as `0.0.0.0`.
+* Anonymized username hashes, which are unique identifiers used solely to identify the number of unique users of the {product-very-short} application.
+
```

Comment:
Web Analytics and System Observability should be aligned. If the other suggestions are not sufficient, add a second empty line here, because:

> Each empty line that precedes the list continuation signals a move up one level of nesting.

https://docs.asciidoctor.org/asciidoc/latest/lists/continuation/#ancestor-list-continuation

```suggestion


```